internal/utils.go

```diff
@@ -487,11 +487,22 @@ func buildPolicyTemplate(
 	configPolicyOptionsOverrides *types.ConfigurationPolicyOptions,
 	configPolicyName string,
 ) map[string]interface{} {
+	var customMessage *types.CustomMessage
+	if policyConf.ConsolidateManifests {
+		customMessage = policyConf.CustomMessage
+	} else {
+		customMessage = configPolicyOptionsOverrides.CustomMessage
+	}
+
 	policySpec := map[string]interface{}{
 		"remediationAction": policyConf.RemediationAction,
 		"severity":          policyConf.Severity,
 	}
 
+	if customMessage != nil {
+		policySpec["customMessage"] = customMessage
+	}
+
```

Comment:
This is oversimplified. It should look more like the `evaluationInterval` conditional further down.